.jshintrc

```diff
@@ -0,0 +1,30 @@
+{
+    "white": true,
+    "trailing": true,
+    "undef": true,
+    "browser": true,
+    "eqeqeq": true,
+    "curly": true,
+    "predef": [
+        "$",
+				"accesstab",
+				"addElementClass",
+				"addLoadEvent",
+        "confirm",
+				"evalJSON",
+				"escape",
+				"excerpt",
+				"forEach",
+				"getCookie",
+				"getElementsByTagAndClassName",
+				"hasElementClass",
+				"hideElement",
+        "jQuery",
+				"log",
+				"removeElementClass",
+				"serializeJSON",
+				"setCookie",
+				"showElement",
+				"unescape"
```

Comment:
I'll still merge this and change it if I ever need to change this file, but mixing tabs+spaces is kind of annoying.